frontend/src/metabase/dashboard/components/EmbedMenu/NonAdminEmbedMenu.tsx

```diff
@@ -27,14 +28,23 @@ export const NonAdminEmbedMenu = ({
 
   const isDisabled = !isPublicSharingEnabled || !hasPublicLink;
 
-  const target = (
-    <ResourceEmbedButton
-      hasBackground={resourceType === "dashboard"}
-      onClick={() => setIsOpen(!isOpen)}
-      disabled={isDisabled}
-      tooltip={isDisabled ? tooltipLabel : null}
-    />
-  );
+  const target =
+    resourceType === "dashboard" ? (
+      <ResourceEmbedButton
+        hasBackground={true}
+        onClick={() => setIsOpen(!isOpen)}
+        disabled={isDisabled}
+        tooltip={isDisabled ? tooltipLabel : null}
+      />
+    ) : (
+      <ViewFooterButton
+        data-testid="resource-embed-button"
+        icon="share"
+        onClick={() => setIsOpen(!isOpen)}
+        disabled={isDisabled}
+        tooltipLabel={isDisabled ? tooltipLabel : null}
+      />
+    );
```

Comment:
It's an inconsistency in styling between the action buttons of the dashboard and question - in the past it was easier since we could just use some styled components to make things work in the `ResourceEmbedButton` component. Now that we're unifying the styles, we can't do that, so I had to make a new component specifically for the question view. I think in the future we'll split these into separate components